### PR TITLE
update before-install script

### PR DESCRIPTION
**Pull request by halfhorst**

### CODE REVIEW

File: ci/travis-before-install.sh [hunk 1]
```diff
@@ -17,29 +17,27 @@ python -V
...
-
-    if [ "${upstream_branch}" ]
+    # if there is a match for upstream, use that
+    if [ ! -z "${upstream_branch_exists}" ]  # checks if not-empty
     then
-        git clone --branch=$branch https://github.com/ihmeuw/${upstream_repo}.git
-        pushd ${upstream_repo}
+        echo upstream branch found for ${branch} in ${upstream_repo}
+        # clone and install upstream stuff if target branch exists
+        git clone --branch=$branch https://github.com/ihmeuw/vivarium.git
```
Comment (kfwilson) [inline ci/travis-before-install.sh]: won't this only install the branch from vivarium? Do you want to be using upstream_repo here?

Comment (halfhorst) [reply ci/travis-before-install.sh]: yeah I just fixed it :grin:

File: ci/travis-before-install.sh [hunk 2]
```diff
@@ -17,29 +17,27 @@ python -V
...
-
-    if [ "${upstream_branch}" ]
+    # if there is a match for upstream, use that
+    if [ ! -z "${upstream_branch_exists}" ]  # checks if not-empty
     then
+        echo upstream branch found for ${branch} in ${upstream_repo}
+        # clone and install upstream stuff if target branch exists
         git clone --branch=$branch https://github.com/ihmeuw/${upstream_repo}.git
-        pushd ${upstream_repo}
+        pushd vivarium
```
Comment (kfwilson) [inline ci/travis-before-install.sh]: should this also be upstream_repo?

Comment (halfhorst) [reply ci/travis-before-install.sh]: yes 100%. I'm slowly getting there :face_with_head_bandage:

Comment (kfwilson) [reply ci/travis-before-install.sh]: haha sorry I kind of jumped the gun. Didn't see your hold message in slack so just charged ahead with reviewing

**Pull request merged**